Lesson Plans/Flight Instructor/CFI/CFI1.4 Teaching Process.md

```diff
@@ -0,0 +1,250 @@
+---
+tags:
+  - cfi
+  - lesson
+---
+Related to [[Steps in the Teaching Process]] and [[Elements of the Teaching Process]]
+
+### Objective
+The learner acquires knowledge and skills related to course & lesson development for aviation instruction.
+
+### Resources
+- [[CFI ACS]] Section FI.I.C
+- [[AIH Ch5]]
+- [[AIH Ch7]]
+
+### Builds On
+None
+
+### Schedule
+Ground 0.5 hours
+
+### Lesson Elements
+
+#### Teaching
+1. Process
+	1. Organizes the material in a way that the learner can understand
+	2. Preparation, Presentation, Application and Assesment
```

Comment:
Spelling on assessment. Also could link or remove [[Steps in the Teaching Process]]